docs/user_guide/fairness_in_machine_learning.rst

```diff
@@ -156,3 +337,22 @@ subpackage :code:`fairlearn.metrics`.
    .. [#3] Agarwal, Beygelzimer, Dudik, Langford, Wallach `"A Reductions
       Approach to Fair Classification"
       <https://arxiv.org/pdf/1803.02453.pdf>`_, ICML, 2018.
+	  
+   .. [#4] Selbst, Andrew D. and Boyd, Danah and Friedler, Sorelle and Venkatasubramanian,
+      Suresh and Vertesi, Janet, "Fairness and Abstraction in Sociotechnical Systems"
+      (August 23, 2018). 2019 ACM Conference on Fairness, Accountability, and Transparency
+      (FAT*), 59-68, Available at `SSRN: 	<https://ssrn.com/abstract=3265913>`_,
+   
+   .. [#5] Mark S. Ackerman. 2000. `"The intellectual challenge of CSCW: The gap between
+      social requirements and technical feasibility"
+      https://doi.org/10.1207/S15327051HCI1523_5`. Human-Computer Interaction 15,
+      2-3 (2000), 179–203.
+
+   .. [#6] Jon Kleinberg, Sendhil Mullainathan, and Manish Raghavan. 2017.
+      `"Inherent trade-offs in the fair determination of risk scores"
+      https://arxiv.org/abs/1609.05807`. In Proc. of ITCS.
+
+   .. [#7] Timnit Gebru, Jamie Morgenstern, Briana Vecchione, Jennifer
+      Wortman Vaughan, Hanna Wallach, Hal Daumé III, Kate Crawford.
+      "Datasheets for Datasets" `https://arxiv.org/abs/1803.09010
+      <https://arxiv.org/abs/1803.09010>`_ (2018)
```

Comment:
Why are these included here? I don't have anything against them (I've only read 6 and 7), but they're not mentioned in the text above. You could add them to this section, though: https://fairlearn.org/main/user_guide/further_resources.html#papers